bin/scripts/lib/fonts.json

```diff
@@ -615,6 +615,17 @@
       "caskName": "victor-mono",
       "repoRelease": true,
       "description": "Clean, crisp and narrow, with a large x-height and clear punctuation"
+    },
+    {
+      "unpatchedName": "Geist Mono",
```

Comment:
You appended this in the bottom, but to keep things accessible the file has the fonts sorted alphabetically.
Better to shift this up.